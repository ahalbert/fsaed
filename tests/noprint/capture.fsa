/beep/ ->
/boop/ {start capture ->} 
/baz/ {stop capture print -> 1}
