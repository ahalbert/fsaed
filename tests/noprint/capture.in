beep
boop - CAPTURED
foo - CAPTURED
bar - CAPTURED
baz
buzz
