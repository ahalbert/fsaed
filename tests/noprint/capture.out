boop - CAPTURED
foo - CAPTURED
bar - CAPTURED
